package.json

```diff
@@ -47,6 +45,5 @@
     "phantomjs-svg2png.js",
     "lru-cache.js",
     "logo"
-  ],
-  "engines": { "node": "0.10.x" }
```

Comment:
I removed the specification which is equivalent to having `"*"` as the specification.  I'm not sure which version is the first to support Promise.  Would you like me to require Node v5?